packages/react/src/components/Tabs/Tabs-test.js

```diff
@@ -241,6 +252,48 @@ describe('Tabs', () => {
           expect(wrapper.state().selected).toEqual(1);
         });
       });
+
+      describe('ignore disabled child tab', () => {
+        const wrapper = mount(
```

Comment:
Any chance we could move `mount` into either a `beforeEach` type block or into a generator function that creates the wrapper? This pattern can cause non-deterministic tests, unfortunately 😞